util/reggen/reg_top.sv.tpl

```diff
@@ -967,14 +1009,6 @@ ${space}\
         ${rd_name}[${str_bits_sv(field.bits)}] = '0;
 % endif
 </%def>\
-<%def name="reg_enable_gen(reg, idx)">\
-  % if reg.needs_re():
-  assign ${reg.name.lower()}_re = addr_hit[${idx}] & reg_re & !reg_error;
-  % endif
-  % if reg.needs_we():
-  assign ${reg.name.lower()}_we = addr_hit[${idx}] & reg_we & !reg_error;
-  % endif
-</%def>\
```

Comment:
This function here was duplicated (there were two exact copies of the same function in this template).